src/schema/common.cljs

```diff
@@ -28,10 +28,15 @@
     [:on-success [:or fn? [:cat keyword? [:* :any]]]]
     [:on-error [:or fn? [:cat keyword? [:* :any]]]]]])
 
+(def ^:private ?exception
+  [:fn {:error/message "schema.common/exception should be of type ExceptionInfo"}
+   (fn [v] (instance? ExceptionInfo v))])
+
 (defn register-schemas
   []
   (registry/register ::theme ?theme)
   (registry/register ::customization-color ?customization-color)
   (registry/register ::public-key ?public-key)
   (registry/register ::image-source ?image-source)
-  (registry/register ::rpc-call ?rpc-call))
+  (registry/register ::rpc-call ?rpc-call)
```

Comment:
we still want/need these?

Reply:
this wasn't added in this PR, but why remove them?

Reply:
well I mean the added schemas here?
- _why remove them?_ - because we agreed yesterday to keep schemas to all things quo and leave other parts of the codebase until quo is finished.